apis/python/tests/test_basic_anndata_io.py

```diff
@@ -219,16 +219,16 @@ def test_resume_mode(adata, resume_mode_h5ad_file):
     tempdir1 = tempfile.TemporaryDirectory()
     output_path1 = tempdir1.name
     tiledbsoma.io.from_h5ad(
-        output_path1, resume_mode_h5ad_file, "RNA", ingest_mode="write"
+        output_path1, resume_mode_h5ad_file.as_posix(), "RNA", ingest_mode="write"
```

Comment:
I think `str(resume_mode_h5ad_file)` might _technically_ be a better way to express this, since there is a theoretical world where tiledbsoma runs on non-POSIX machines. `str(some_pathlib_path)` turns it into the system’s default representation. (I’m pretty sure I have myself written `.as_posix()` elsewhere in tiledbsoma without thinking that part through, so it basically Does Not Matter.)